File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.java
// Last updated: 7/9/2026, 7:40:51 AM
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int X = 0;
        for(int i = 0; i < operations.length; i++){
            if(operations[i].equals("--X")){
                X = X - 1;
            }
            if(operations[i].equals("X--")){
                X = X - 1;
            }
            if(operations[i].equals("X++")){
                X = X + 1;
            }
            if(operations[i].equals("++X")){
                X = X + 1;
            }
        }
        return X;
    }
}

File: 2137-FinalValueOfVariableAfterPerformingOperations/2137-FinalValueOfVariableAfterPerformingOperations.py
# Last updated: 7/9/2026, 7:40:56 AM
class Solution(object):
    def finalValueAfterOperations(self, operations):
        """
        :type operations: List[str]
        :rtype: int
        """
        X = 0
        for op in operations:
            if '+' in op:
                X += 1
            else:
                X -= 1
        return X

        
        